lib/lz4hc.h

```diff
@@ -405,6 +405,18 @@ LZ4LIB_STATIC_API void LZ4_attach_HC_dictionary(
           LZ4_streamHC_t *working_stream,
     const LZ4_streamHC_t *dictionary_stream);
 
+/*! LZ4_getDictHCSize():
```

Comment:
I presume this entry point is not needed anymore

Reply:
Sorry I forgot to remove that.